pinot-segment-local/src/main/java/org/apache/pinot/segment/local/upsert/merger/PartialUpsertColumnarMerger.java

```diff
@@ -74,7 +75,10 @@ public void merge(LazyRow previousRow, GenericRow newRow, Map<String, Object> re
       // (1) If the value of the previous is null value, skip merging and use the new value
       // (2) Else If the value of new value is null, use the previous value (even for comparison columns)
       // (3) Else If the column is not a comparison column, we applied the merged value to it
-      if (!(merger instanceof OverwriteMerger)) {
+      if (merger instanceof ForceOverwriteMerger) {
```

Comment:
please add some comment Similar to Overwrite mergers.
If the merge strategy is force overwrite merger, always use the new value no matter what it's.
The idea is to minimize the access to the previous row value to reduce disk IO.

Reply:
Added comments